addon/styles/_colors.scss

```
@@ -1,5 +1,162 @@
-// See https://zeroheight.com/8dd127da7/p/839645-couleurs/b/21317a
+// See https://www.figma.com/file/8RJ3aCSfdeQ8AZZVBBYKS8/Design-System-Pix?node-id=16%3A2
 
+//// SOLID
+// Primary
+$pix-primary: #3D68FF;
+$pix-primary-80: #000E87;
+$pix-primary-70: #0D25B3;
+$pix-primary-60: #2044DC;
+$pix-primary-40: #5B94FF;
+$pix-primary-30: #88BEFF;
+$pix-primary-20: #C4E6FF;
+$pix-primary-10: #DCF1FF;
+$pix-primary-5: #F2F9FF;
+
+// Secondary
+$pix-secondary: #FFD235;
+$pix-secondary-80: #874D00;
+$pix-secondary-70: #A95800;
+$pix-secondary-60: #CE8900;
+$pix-secondary-50: #EAA600;
+$pix-secondary-40: #EAA600;
+$pix-secondary-20: #FFE381;
+$pix-secondary-10: #FFEEB3;
+$pix-secondary-5: #FFF9E6;
+
+// Secondary Certif
+$pix-secondary-certif: #20B4AF;
+$pix-secondary-certif-80: #093432 ;
+$pix-secondary-certif-70: #0E4D4C ;
+$pix-secondary-certif-60: #126765;
+$pix-secondary-certif-50: #17817E;
+$pix-secondary-certif-30: #34DBD6;
+$pix-secondary-certif-20: #67E4E0;
+$pix-secondary-certif-10: #9AEDEB;
+$pix-secondary-certif-5: #CCF6F5;
+
+// Secondary Orga
```

J'ai vu plus bas Pix App, si ceux là concerne les apps Certif et Orga, autant mettre App aussi ? 🙂

```suggestion
// Secondary Pix Certif
$pix-secondary-certif: #20B4AF;
$pix-secondary-certif-80: #093432 ;
$pix-secondary-certif-70: #0E4D4C ;
$pix-secondary-certif-60: #126765;
$pix-secondary-certif-50: #17817E;
$pix-secondary-certif-30: #34DBD6;
$pix-secondary-certif-20: #67E4E0;
$pix-secondary-certif-10: #9AEDEB;
$pix-secondary-certif-5: #CCF6F5;

// Secondary Pix Orga
```